themes/default/content/registry/packages/aws-native/_index.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: AWS Native
-meta_desc: Learn how you can use Pulumi's AWS Native Provider to reduce the complexity of provisioning and managing resources on AWS.
+title: AWS Native Cloud Control
+meta_desc: Learn how you can use Pulumi's AWS Native Cloud Control Provider to reduce the complexity of provisioning and managing resources on AWS.
 layout: package
 aliases:
     - "/docs/reference/clouds/aws-native/"
     - "/docs/intro/cloud-providers/aws-native/"
 ---
 
 {{% notes type="info" %}}
-AWS Native is in public preview. AWS Native provides coverage of all resources in the [AWS Cloud Control API](https://aws.amazon.com/blogs/aws/announcing-aws-cloud-control-api/), including same-day access to all new AWS resources. However, some AWS resources are not yet available in AWS Native.
+AWS Native Cloud Control is now Generally Available. AWS Native Cloud Control provides coverage of all resources in the [AWS Cloud Control API](https://aws.amazon.com/blogs/aws/announcing-aws-cloud-control-api/), including same-day access to all new AWS resources. However, some AWS resources are not yet available in AWS Cloud Control.
```

Comment:
Maybe just drop the first sentence and start with the description (GA things don't need special prefaces about their status).